modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -104,7 +104,7 @@ By default, database metrics include:
 |<prefix>.check_point.events|The total number of checkpoint events executed so far. (counter)
 |<prefix>.check_point.total_time|The total time, in milliseconds, spent in checkpointing so far. (counter)
 |<prefix>.check_point.duration|The duration, in milliseconds, of the last checkpoint event. Checkpoints should generally take several seconds to several minutes. Long checkpoints can be an issue, as these are invoked when the database stops, when a hot backup is taken, and periodically as well. Values over `30` minutes or so should be cause for some investigation. (gauge)
-|<prefix>.check_point.flushed_bytes|label:new[Introduced in 5.10]The accumulated number of bytes flushed during the last checkpoint event. (gauge)
```

Comment:
So its accumulated because thats like for all checkpoints combined and not the only one.
Plus its gauge as i can see in the code

Reply:
good, so if the docs are correct, then we have to update the code